bloom-instance/s3.tf

```diff
@@ -1,3 +1,40 @@
 resource "aws_s3_bucket" "user_upload_bucket" {
   bucket_prefix = "${var.name_prefix}-user-uploads"
 }
+
+resource "aws_s3_bucket_cors_configuration" "user_upload_cors_config" {
+  bucket = aws_s3_bucket.user_upload_bucket.id
+
+  cors_rule {
+    allowed_headers = ["*"]
+    allowed_methods = ["PUT", "GET"]
+    allowed_origins = ["http://localhost:3001"]
```

Comment:
This should not be a hardcoded value.  It would not, for example, work with the services currently running in our AWS environment.  It should be enough to just add a var for this of type `list(string)`.